src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -106,21 +90,63 @@ def get_metadata(self) -> dict:
         return metadata
 
     def get_original_timestamps(self) -> np.ndarray:
-        raise NotImplementedError(
-            "Unable to retrieve the original unaltered timestamps for this interface! "
-            "Define the `get_original_timestamps` method for this interface."
-        )
+        raise NotImplementedError("The AudioInterface does not yet support timestamps.")
 
-    def get_timestamps(self) -> np.ndarray:
-        raise NotImplementedError(
-            "Unable to retrieve timestamps for this interface! Define the `get_timestamps` method for this interface."
-        )
+    def get_timestamps(self) -> Optional[np.ndarray]:
+        raise NotImplementedError("The AudioInterface does not yet support timestamps.")
+
+    def align_timestamps(self, aligned_timestamps: List[np.ndarray]):
+        raise NotImplementedError("The AudioInterface does not yet support timestamps.")
+
+    def align_starting_time(self, starting_time: float):
```

Comment:
I feel that the term `starting_time` is overloaded here (and maybe in other instances of this?). 

As in, you are referring to `starting_time` as the time when this stream / device / recorder started acquiring data with respect to the start of the session. That is, this `starting_time` is relative to the session. But, if I understand correctly, you also use the term `starting_time` for the segments relative to the stream / device / recorder. That is, those `starting_times` are relative to the themselves.

I kind of feel that we should have the concept of session time here. That said, you probably have thought more about this. How do you feel? am I missing something?

Reply:
> As in, you are referring to starting_time as the time when this stream / device / recorder started acquiring data with respect to the start of the session.

That is the intention, yes.


> But, if I understand correctly, you also use the term starting_time for the segments relative to the stream / device / recorder. That is, those starting_times are relative to the themselves.

Can you point to where you're looking for the segment specific stuff? Some internal code might be irrelative to the session start time, but the methods should generally be w.r.t. the session start time


> I kind of feel that we should have the concept of session time here. That said, you probably have thought more about this. How do you feel? am I missing something?

Kind of tricky, because knowing the exact session start time as, say, a `datetime` would not be helpful if we knew the `datetime` of each interface (recorded on separate device, whose clocks are entirely out of sync). Instead, all that can be known by alignment is the relative time as it occurs on a centralized device, usually as sent by TTLs from a secondary system

Reply:
I think that I was confused because I was thinking in multisegments objects from spikeinterface. Those objects have starting times `t_start` for every segment and I think they are relative to the first segment. But now that I think about it, I am not sure this is even true for all modalities in spikeinterface.

Reply:
>  I am not sure this is even true for all modalities in spikeinterface.

Uhh I hope so lol, can you confirm that we can trust that? lol

We used that mechanism for our temporal alignment methods here for multi-segment recording/sorting interfaces

I essentially sought to mimic it here viewing each separate file as a different 'segment'

Reply:
That's my belief as well but I don't know. I will confirm with Sam in our next meeting.